### ISSUE TITLE
Sync between mobile and PC (file storage)

### ISSUE THREAD

**Issue by altwarg** [laurent22/joplin#5336]
## Environment

Joplin version: 2.2.7 on Windows and 2.3.4 on Android
Platform: Windows 10 + Android 11

## Steps to reproduce

1. Set the synchronization target to file storage on PC
2. Create some notes in notebooks with nested structure, i.e.:
![image](https://user-images.githubusercontent.com/34391398/129918014-707bcb49-0493-4746-a76f-50e97db2fc6e.png)

3. Copy folder with notes on Android device
4. Set the synchronization target to file storage on Android
5. Try to sync notes -- some notes in nested notebook (as well as these sub-notebooks) are missing:
![image](https://user-images.githubusercontent.com/34391398/129918223-a8554c2e-a15e-4100-8bc5-614fd31833f2.png)

**Comment by altwarg** [comment]
Any updates on the issue?

**Comment by ManApart** [comment]
I had this issue because  I had upgraded my mobile version and that broke the sync, and I didn't realize I needed to resync.

I couldn't find an easy way to redo the sync on mobile. In the end, I deleted the app, re installed it, and re-setup sync, and then things worked for me. Mobile was 2.3.4 and desktop was 2.3.5

**Comment by altwarg** [comment]
@ManApart, yes, the situation you described is also another issue, but I have problems that part of my notes are just disappeared after syncing on phone.

**Comment by Daeraxa** [comment]
I wouldn't really look into any of it until your sync targets match. So long as data is still correct on one of your devices, in a worst case scenario you can still re-sync everything if something has gone wrong.

https://joplinapp.org/changelog/#v2-3-3-https-github-com-laurent22-joplin-releases-tag-v2-3-3-2021-08-14t09-19-40z
>v2.3.3 - 2021-08-14T09:19:40Z🔗
CAUTION: This release will ask you to upgrade your sync target, whether it's Dropbox, OneDrive, Joplin Cloud, etc. Once it is done, only apps version 2.3+ will be able to sync with it, so make sure you are ready to upgrade all your apps before installing this version.
